pkg/operator/audit/audit.go

```diff
@@ -0,0 +1,104 @@
+package audit
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	kyaml "k8s.io/apimachinery/pkg/util/yaml"
+	auditv1beta1 "k8s.io/apiserver/pkg/apis/audit/v1beta1"
+
+	"github.com/openshift/cluster-openshift-apiserver-operator/pkg/operator/v311_00_assets"
+	libgoapiserver "github.com/openshift/library-go/pkg/operator/configobserver/apiserver"
+)
+
+const (
+	policyAsset = "v3.11.0/openshift-apiserver/audit-policies-cm.yaml"
+)
+
+func NewAuditPolicyPathGetter() (libgoapiserver.AuditPolicyPathGetterFunc, error) {
```

Comment:
Can we share this in library-go?

Reply:
yes, working on a follow up library-go pr to share the static asset and the above function. This will require changes to `staticresourcecontroller.NewStaticResourceController`.